docs/troubleshooting.md

```diff
@@ -10,9 +21,8 @@
 
    If this returns an error, make sure you have Go installed and your Go environment is setup. Then redo [Step 1](#basic-usage---relaying-packets-across-chains).
 
----
 
-## **Verify valid `chain`, `client`, and `connection`**
+### **Verify valid `chain`, `client`, and `connection`**
```

Comment:
L25 and L36 are identical. It seems like this line may be supposed to read something along the lines of, "Verify valid `Chain`, `Keys`, `Balance` and `Path`